src/TurbulenceClosures/isopycnal_rotation_tensor_components.jl

```diff
@@ -1,32 +1,59 @@
 # tracer components of the Redi rotation tensor
 
-@inline function isopycnal_rotation_tensor_xz_fcc(i, j, k, grid::AbstractGrid{FT}, buoyancy, tracers) where FT
+"""
+    AbstractTurbulenceClosure
+
+Abstract supertype for turbulence closures.
+"""
+abstract type AbstractIcopycnalModel end
+
+struct SlopeApproximation <: AbstractIcopycnalModel end
```

Comment:
Here's an idea --- we can use `isopycnal_model=nothing` as an abstraction for "no model", eg the "full" isopycnal slope... ?

Reply:
Although this prevents us from adding a `slope_limiter` property so maybe not a good idea.

Can we come up with a better name that means basically "full rotation tensor without approximation"...?

Reply:
I didn't like the `SlopeApproximation` name either... I wrote `FullSlopeApproximation` initially but then thought that was stupid :)

Reply:
I think something like `FullRotationTensor` or `NoApproximation`... ?

Reply:
Or `NonlinearRotationTensor` (contrasted with `LinearizedRotationTensor`, see below)